hardware/products/ucm/uc5550.md

```diff
@@ -0,0 +1,257 @@
+# UC5550
+![G400D](images/g400d.jpg)
+# Overview
+---
+The UC5550 is a UCM that supports the STM F4 and F7 processor with 16MB of external RAM. It has a built-on STM WiFi module and an Ethernet PHY. It also has 4MB of external flash for large user programs.
+
+# Specifications
+---
+|					|							|
+| -----------------	| -------------------------	|
+| Processor			| STM 32-bit ARM Cortex-M4	|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+
+# Resources
+---
+## Documents
+* [Schematic](http://files.ghielectronics.com/downloads/Schematics/Systems/G400D%20Dev%20Board%20Schematic.pdf)
```

Comment:
This shouldn't be for the dev board.

Reply:
These are just placeholders. The real stuff will be added if this is approved for the layout.

Reply:
Same note as above, set it to the url it will be so we get a build warning so we don't forget.

Reply:
Ok.